### ISSUE TITLE
file's filename cannot be edited

### ISSUE THREAD

**Issue by amnonkhen** [ebi-ait/dcp-ingest-central#442]
Per @Wkt8 's testing for the import/export feature:

file's filename cannot be edited. This should be a ticket for a new feature. However, it is my understanding that no error message occurs when changing the file's filename via spreadsheet. If it does not take too long, a task for showing an error message should be prioritised. Otherwise low priority until we have evidence of a direct project being blocked by it.
